tests/integration/targets/k8s_apply/tasks/main.yml

```diff
@@ -449,326 +450,14 @@
         that:
           - deploy_after_serviceaccount_removal is failed
 
-    - name: Insert new service port
```

Comment:
Why did these get deleted?

Reply:
all this is a duplicate test starting at line 206, seems like a wrong copy/paste